sbt-dump-event-schema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/CrossVersionReporter.scala

```diff
@@ -0,0 +1,19 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.reporters.Reporter
+
+object CrossVersionReporter {
+  def apply(global: Global): Reporter = {
+    val ru = scala.reflect.runtime.universe
+    val rm = ru.runtimeMirror(getClass.getClassLoader)
+    val instanceMirror = rm.reflect(global)
+    val reporterSymbol = ru.typeOf[Global].decl(ru.TermName("reporter"))
+    val reporter =
```

Comment:
Just so that I understand the purpose of this class... is there a problem then in some versions of Scala `reporter` is a method and in others it's a field? Pls specify in a code comment what's the exact problem you're solving here